test/utils/fake.js

```diff
@@ -14,20 +14,23 @@ const crowdsaleData = ({ wallet, overpaymentWallet, token, ...fields }) => {
   // instead we need the rate to represent how many wei one token costs.
   const rate = new BigNumber(1)
 
-  // usually this cap is represented in wei but we need it to be in USD.
-  const cap = toWei(100)
+  // the default goal is represented in USD cents.
+  const goal = fields.goal || new BigNumber(1000000000)
 
-  // usually the goal is represented in wei but we need it to be in USD.
-  const goal = toWei(50)
+  // the default cap is 3 times the goal.
+  const cap = fields.cap || goal.times(3)
 
-  // just set a default dollar rate of half an ETH
-  const dollarRate = toWei(0.5)
+  // as a default let's say
+  // current ETH rate is approx USD 850
```

Comment:
const ethPerUsd = 1 / 850
const ethPerCent = ethPerUsd / 100
const usdConversionRate = toWei(ethPerCent)
Not sure if this^ structure would help with additional tests